### PR TITLE
ToCode Solution: תרגול: מבוא לשורת הפקודה

### PR DESCRIPTION
**Pull request by Nadav1234**

### CODE REVIEW

File: 07_cmd_lab/cmdlab.txt [hunk 1]
```diff
@@ -2,14 +2,26 @@ Intro To Command Line Lab.
...
-  - your current shell?
+  - your current shell? 
   - your default shell?
   - a list of all shells currently installed on the machine?
 
+
+-echo $0
+-echo $SHELL
+-cat /etc/shells
+
```
Comment (ynonp) [inline 07_cmd_lab/cmdlab.txt:13]: 👍

File: 07_cmd_lab/cmdlab.txt [hunk 2]
```diff
@@ -2,14 +2,26 @@ Intro To Command Line Lab.
...
 עברית שפה קשה. בעקרון המעטפת הנוכחית שלי היא bash
 
 2. View the file /etc/passwd on your machine with the command "cat /etc/passwd". Then read about the file in "man 5 passwd". What information is held in the file? What are the fields in each line?
 
+-Contains all users passwords in the following format:
+login name:optional encrypted password:numerical user ID:numerical group ID:user name or comment field:user home directory:optional user command interpreter
+
```
Comment (ynonp) [inline 07_cmd_lab/cmdlab.txt:20]: Well it does contain user data, but no password is stored there for a few decades :) You can find the actual encrypted passwords in `/etc/shadow`

File: 07_cmd_lab/cmdlab.txt [hunk 3]
```diff
@@ -2,14 +2,26 @@ Intro To Command Line Lab.
...
 3. In Unix, the /etc folder holds configuration files. Display all files in "/etc" folder ordered by file size (Hint: search in "man ls")
 
+ls -s /etc
```
Comment (ynonp) [inline 07_cmd_lab/cmdlab.txt:23]: That should be `ls -S /etc`. To actually see it working though you'll want to add -l, so try: `ls -lS /etc`

File: 07_cmd_lab/cmdlab.txt [hunk 4]
```diff
@@ -2,14 +2,26 @@ Intro To Command Line Lab.
...
+
 4. Print current date/time formatted as: "Sun 21/03/2010, 14:30". Hint: read "man date" and search for the correct format string.
+
+date -R
```
Comment (ynonp) [inline 07_cmd_lab/cmdlab.txt:27]: Didn't work on my machine. Can you paste the output `date -R` printed? It should be formatted as `Sun 23/03/2010 14:30` (i.e. first a day, then the date dd/mm/yyyy and then the time HH:MM)

**Review by ynonp — CHANGES_REQUESTED** [review]
Please add a working solution for the last question

**Comment by Nadav1234** [comment]
>  4. Print current date/time formatted as: "Sun 21/03/2010, 14:30". Hint:
read "man date" and search for the correct format string.
+
+date -R
Didn't work on my machine. Can you paste the output date -R printed? It
should be formatted as Sun 23/03/2010 14:30 (i.e. first a day, then the
date dd/mm/yyyy and then the time HH:MM)

$date +"%A %d/%m/%Y %R"
Not quite sure if there's supposed to be a shorter option..

**Comment by ynonp** [comment]
This one worked well.

**Pull request closed without merging**